docs/docs/Integrations/integrations-instana-traceloop.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Traceloop
+slug: /integrations-instana-traceloop
+description: Instrument Langflow with the Traceloop SDK and export traces and metrics to Instana using OpenTelemetry.
+---
+
+Traceloop SDK is a lightweight instrumentation toolkit designed for LLM (Large Language Model) applications. It enables developers to automatically capture and export traces, metrics, and key observability signals from their LLM-powered workflows.
+
+When combined with Instana, the exported telemetry data from Traceloop provides end-to-end visibility, allowing users to visualize traces, analyze performance bottlenecks, and ensure reliable operation of LLM-driven applications.
+
+This guide demonstrates how to integrate the Instana observability platform with your Langflow application using the Traceloop SDK so you can monitor and analyze LLM performance.
+
+## Before you begin
+
+- Create a [Traceloop API key](https://app.traceloop.com/settings/api-key)
+- Create an [Instana endpoint and Instana key](https://www.ibm.com/docs/en/instana-observability/1.0.302)
+- [Install and start Langflow](/get-started-installation)
+
+## Configure environment variables
+
+### Step 1: Create environment file
+Create a `.env` file in your Langflow application root directory.
+
+### Step 2: Add Traceloop API key
+```dotenv
+TRACELOOP_API_KEY=<your_traceloop_api_key>
+```
+This variable authenticates your application with Traceloop's monitoring service.
+
+### Step 3: Set base URL
+```dotenv
+TRACELOOP_BASE_URL=<instana_endpoint>
+```
+This variable configures the Instana endpoint for telemetry data. Example: `https://otlp-magenta-saas.instana.rocks:4318`
+
+### Step 4: Add Instana headers
+```dotenv
+TRACELOOP_HEADERS="x-instana-key=<your_instana_key>"
+```
+This variable provides authentication for Instana data collection.
+
+### Step 5: Configure OpenTelemetry Protocol(OTLP) security
+Check your Instana endpoint URL from Step 3.
+#### For secure connections (HTTPS/TLS)
+```dotenv
+OTEL_EXPORTER_OTLP_INSECURE=false
+```
+
+#### For insecure connections (HTTP, local development)
+```dotenv
+OTEL_EXPORTER_OTLP_INSECURE=true
+```
+For production Instana SaaS endpoints (like https://otlp-magenta-saas.instana.rocks:4318), typically use `false`
+
+### Step 6: Set metrics endpoint
+```dotenv
+TRACELOOP_METRICS_ENDPOINT=host.docker.internal:8000
+```
+This variable configures a separate metrics endpoint (typically for Docker environments).
+
+### Step 7: Enable metrics collection
+```dotenv
+TRACELOOP_METRICS_ENABLED=true
+OTEL_METRIC_EXPORT_INTERVAL=10000
+```
+These variables enable metrics collection with 10-second export intervals.
+
+### Step 8: Configure OpenTelemetry Collector
+Make sure the OpenTelemetry (OTEL) Collector is running. In your Collector's `config.yaml`, add the following values:
+
+```yaml
+# Example values — update to your environment
```

Comment:
We generally tell users to replace the values outside of the code sample.
For example, see [Step 2](https://docs.langflow.org/mcp-tutorial#create-a-python-application-that-connects-to-langflow)